global_config.json

```diff
@@ -3,6 +3,7 @@
   // "token": "<REPLACE ME>", // The auth token to access Answers experience.
   // "apiKey": "<REPLACE ME>", // The answers api key found on the experiences page. This will be provided automatically by the Yext CI system
   // "experienceVersion": "<REPLACE ME>", // the Answers Experience version to use for API requests. This will be provided automatically by the Yext CI system
+  // "environment": "<REPLACE ME>", // The environment to run on for this Answers Experience. (e.g. 'production' or 'sandbox')
```

Comment:
maybe `Must be 'production' or 'sandbox'`? do we support other options?

Reply:
Or even just `i.e.` instead of `e.g.`

Reply:
hmm from the looks of it (i.e. in `urlutils.js`), I only know we have production and sandbox endpoints. So I can update the comment as you suggested.